pkgs/development/python-modules/PyInstaller/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchPypi, python3Packages }:
+
+python3Packages.buildPythonPackage rec {
+  pname = "PyInstaller";
+  version = "4.3";
+
+  src = fetchPypi {
+    pname = "pyinstaller";
+    inherit version;
+    sha256 = "sha256-Xs+LvCMNcpinluUrt0W5Xu4Sh40UHxZFYSyZJG7NI/I=";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    altgraph
+    pyinstaller-hooks-contrib
+    macholib
+  ];
+
+  doCheck = false;
```

Comment:
pythonImportsCheck is required when disabling tests and also add a comment why you disabled tests.